xml/inst_yast2.xml

```diff
@@ -2392,7 +2392,17 @@ sle_update.xml#sec.update.terminology: "Modules" -->
     </listitem>
    </itemizedlist>
   </note>
-
+  <note>
+   <title>Mount Points for FCoE and iSCSI Devices</title>
+   <para>FCoE and iSCSI devices will appear asynchronously during the
+    boot process. While the initrd guarantees that those devices are
+    setup correctly for the root file system, there are no such
+    guarantees for any other file systems or mount points like 
+    <filename>/usr</filename>. Hence any system mount points like 
+    <filename>/usr</filename> or <filename>/var</filename> are not 
+    supported. If you want to use those devices, ensure correct
+    synchronisation with the respective services and devices.</para>
```

Comment:
Sounds a bit circular: "to use devices ... ensure sync with devices." - not sure here.